src/index.html

```diff
@@ -0,0 +1,46 @@
+<!-- <!DOCTYPE html> -->
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+
+    <title>Weather App</title>
+    <link href="https://cdn.jsdelivr.net/gh/gitbrent/bootstrap-switch-button@1.1.0/css/bootstrap-switch-button.min.css"
```

Comment:
- [ ] Kindly delete this index.html file under the src directory since it serves no purpose.

Reply:
It gets compiled by webpack to produce the html file located at dist.